ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml

```diff
@@ -64,12 +64,22 @@ gcc73_devtoolset_compiler:
   - devtoolset-7-binutils         # OpenJ9
   - devtoolset-7-gcc-c++          # OpenJ9
 
+c_lib_32bit_CentOS_x86_64:
+  - glibc.i686
+  - glibc-devel.i686
+  - libstdc++.i686
+
 Additional_Build_Tools_CentOS7:
```

Comment:
It looks like you have this doubled up. `c_lib_32bit_CentOS_x86_64` and `C_Library_32bit_CentOS_x86_64`

Reply:
Thank you!